time t
int32 task
duration deltat
int32 direction
float64 phase
float64 limitL
float64 limitR
